browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html

```diff
@@ -28,6 +45,35 @@
         </div>
       </div>
     </div>
+    <settings-toggle-button id="advancedViewControlType"
+        pref="{{prefs.advanced_view_default}}"
```

Comment:
`prefs.brave.advanced_view_default` (missing the `.brave.` though you have it in the commented out code)

And you don't need a `on-settings-boolean-control-change` handler since this is a straightforward boolean mapping, though you'll need to do custom mapping for the commented out dropdown ;-)